ffv1.md

```diff
@@ -1014,9 +1021,13 @@ Parameters( ) {                                               |
     colorspace_type                                           | ur
     if (version >= 1)                                         |
         bits_per_raw_sample                                   | ur
-    chroma_planes                                             | br
-    log2_h_chroma_subsample                                   | ur
-    log2_v_chroma_subsample                                   | ur
+    if (colorspace_type == 2) {                               |
+        rggb_order                                            | ur
+    } else {                                                  |
+        chroma_planes                                         | br
+        log2( h_chroma_subsample )                            | ur
```

Comment:
This requires a global change of `log2_h_chroma_subsample` to `log2( h_chroma_subsample )`, same for vertical. Also this approach would require a version bump, so version 4, else an existing version 3 decoder would interpret these Parameters incorrectly, so I'd suggest a `if (version >= 4)` outside of the `if (colorspace_type == 2)`